plugins/ipam/host-local/backend/disk/lock.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
```

Comment:
Just do !windows here - this "should" work on all Unix-like systems.

Reply:
Done